src/fight.cpp

```diff
@@ -5705,18 +5705,37 @@ void roll_individual_initiative(struct char_data *ch)
 {
   if (AWAKE(ch))
   {
+    // this also applies wound modifiers
     GET_INIT_ROLL(ch) = roll_default_initiative(ch);
-    GET_INIT_ROLL(ch) -= damage_modifier(ch, buf, sizeof(buf));
+
     if (AFF_FLAGGED(ch, AFF_ACTION)) {
       GET_INIT_ROLL(ch) -= 10;
       AFF_FLAGS(ch).RemoveBit(AFF_ACTION);
     }
+
+    // By the book, MBW 3 adds an extra action to their first pass and 4 adds an additional action to their second.
+    // This implementation is mechanically weaker since chars will get the extras at the end of the turn.
+    for (obj_data *cyber = ch->cyberware; cyber; cyber = cyber->next_content) {
+      if (GET_CYBERWARE_TYPE(cyber) == CYB_MOVEBYWIRE) {
+        if (GET_CYBERWARE_RATING(cyber) == 3)
+          GET_INIT_ROLL(ch) += 10;
```

Comment:
+10 is a full extra pass, how about +5 for 3 and +10 for 4?

Reply:
Yeah, a full pass / 2 full passes are intentional, to actually allow the additional attack / 2 attacks as seems to be intended by the book. Part of the more generous reading is to try to get players to actually consider this 'ware as a feasible option.

Though maybe the potential for 2 more rounds of "out of initiative" messages outweighs that, so I'll tone that down.